### PR TITLE
Robot localization

### PR DESCRIPTION
**Pull request by Samuel-Moore-1714**

### CODE REVIEW

File: src/main/java/frc/robot/SwerveModule.java [hunk 1]
```diff
@@ -97,11 +97,6 @@ public double makePositiveDegrees(Rotation2d anAngle){
         return makePositiveDegrees(anAngle.getDegrees());
     }
 
-    public Rotation2d getAngle(){
```
Comment (Dylan-Powers) [inline src/main/java/frc/robot/SwerveModule.java:100]: Why get rid of getAngle()?

File: src/main/java/frc/robot/commands/LockAprilTag.java
Comment (Dylan-Powers) [inline src/main/java/frc/robot/commands/LockAprilTag.java:1]: Are we still going to use this command? Looks unfinished.

File: src/main/java/frc/robot/SwerveModule.java (hunk 1 above)
Comment (Samuel-Moore-1714) [reply src/main/java/frc/robot/SwerveModule.java:100]: It uses a deprecated method, and I was trying to figure out how to replace it when I realized that it is never used anywhere in our code ever. If we need to add it back we can do so later.

File: src/main/java/frc/robot/commands/LockAprilTag.java
Comment (Samuel-Moore-1714) [reply src/main/java/frc/robot/commands/LockAprilTag.java:1]: No, we are not going to use it. This was made before we started using robot localization in the swerve subsystem. We can delete it.

**Pull request closed without merging**